src/Protocol/ActivityPub/Processor.php

```diff
@@ -298,6 +298,9 @@ public static function followUser($activity)
 		$item = ['author-id' => Contact::getIdForURL($activity['actor']),
 			'author-link' => $activity['actor']];
 
+		// Ensure that the contact has got the right network type
+		self::switchContact($item['author-id']);
```

Comment:
Shouldn't it be `Receiver::switchContact()`?

Reply:
This is potentially confusing to anyone who isn't you, can you please be abundantly clear about the differences in the method doc blocks? Like `This method do X, if you want to do Y, use the other one instead`.

Reply:
This one is a local one. So you cannot mix them.

Reply:
Gotcha. I've seen you expanded the scope of the other one, so I assumed this one was public too.

Reply:
I really fought with myself, if I should make a single function or name them differently. But they are only doing **nearly** the same, so mixing them would blow them up.

Reply:
I'm surprised you didn't go with the usual "Add an obscure boolean flag parameter to make the method do two different things"! 😄

Reply:
Although you might not believe me, but sometimes old dogs **can** learn new tricks. 😬